keanu-project/src/test/java/io/improbable/keanu/tensor/dbl/Nd4jTensorTestHelpers.java

```diff
@@ -44,4 +44,14 @@ public static void assertMinusInPlaceOperationEquals(DoubleTensor left, DoubleTe
         assertEquals(expected, left);
     }
 
+    public static void assertPowOperationEquals(DoubleTensor left, DoubleTensor right, DoubleTensor expected) {
+        DoubleTensor actual = left.pow(right);
+        assertEquals(expected, actual);
+    }
+
+    public static void assertPowInPlaceOperationEquals(DoubleTensor left, DoubleTensor right, DoubleTensor expected) {
+        left = left.powInPlace(right);
+        assertEquals(expected, left);
+    }
+
```

Comment:
these methods are always called in pairs (the in-place and non-in-place versions). I think you can combine the two methods together. It would feel like they had more purpose then too, since currently they aren't gaining you much compared to just inlining them.